### ISSUE TITLE
explain better the logic behind "Inventory Alert Threshold"

### ISSUE THREAD

**Issue by fabricat** [grokability/snipe-it#9536]
Reading the documentation about [Email Alerts](https://snipe-it.readme.io/docs/email-alerts), it was not clear to me how **Inventory Alert Threshold** is combined with the **Minimum Quantity** set on each accessory/consumable/component.

In practice, the Threshold value is **added** to the Min.Qty, in order to calculate the "almost below minimum quantity levels".
Is that correct?

**Comment by KayGt3** [comment]
Yes, when you reach **Minimum quantity + Inventory Alert Threshold** you wil notified about that.

**Comment by fabricat** [comment]
Thanks for the confirmation @KayGt3 👍 

Do you agree that it is better to make that clear on the [documentation page](https://snipe-it.readme.io/docs/email-alerts)?

**Comment by KayGt3** [comment]
You are welcome!

I agree with you, it should be more clear but anyway with some test you can understand how it works. :wink:
